facdb/sql/dsny_textiledrop.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS _dsny_textiledrop;
+SELECT uid,
+    source,
+    vendor_name||' '||'Textile Drop-off Site' as facname,
```

Comment:
Why use pipes (||) here instead of `CONCAT` as you have for other scripts?
Also, should we update this to `CONCAT(vendor_name, ': DSNY Textile Drop-off Site') as facname`

Reply:
No, we should not update this to vendor_name, ': DSNY Textile Drop-off Site' as facname.
Outstanding question, Why use pipes (||) here instead of CONCAT as you have for other scripts?